# Scientific Researcher / aspiring Data Scientist
Programing Skills: 
Python,R, SQL, UCSF ChimeraX, PyMOL, Maxquant, Perseus, pLINK and Xi

Technical Skills:
LC-MS/MS, HPLC, affinity purification, shotgun-, phosphoproteomics techniques

## Work Experience
**Scientific Researcher @ Wageningen University and Research (2019 - present)**
 * Consulted on various proteomics and data projects
 * Provided data analytics solutions to diverse omics datasets
 * Supervised students, PhD and Post-Docs

**Graduate student @ Wageningen University and Research (2015-2019)**
* Implemented and developed technical approaches for state of the art mass spectrometry based proteomics
* Used several data analytics approaches to reveil patterns and insights in several omics related projects

**MSc. Thesis and internship @ Radboud institute of molecular life sciences (2014-2015)**
* Developed a minimal tag enrichment approach using bioorthogonal chemistry and amber suppression technology

**BASc. Thesis and internship @ Murdoch Childrens reserach Institute, Melbourne, AU (2012-2013)**
* Used diverse molecular biological techniques to investigate β-hemoglobinopathies

## Education
* PhD.,  Biochemistry/Proteomics, Wageningen University & Research  2020
* MSc.,  Biochemistry with a focus on Statistics and Bioinformatics, Wageningen University & Research  2015
* BASc., Biotechnology,Van Hall Larenstein University of Applied Sciences 2013

## Certificates/Accomplishments
* IBM Data Analyst Specialization (2023)
* IBM Data Science Specialization (2023)
* The Data Science Course: Complete Data Science Bootcamp (2023)

## Projects
**Predicting subcellular locations of proteins in A.Thaliana**

_Project status Ongoing._

Using dynamic organellar mapping workflows the subcellular location and movement of proteins are predicted by applying machine learning algorithms and Bayesian inference using **Pythons scikit-learn**  and **PyMC3** packages.The outcome of this project allows researchers to identify moving proteins in a holistic approach between different experimental conditions.

![Dynamicorganellarmaps](/assets/img/Organellarmaps.jpg)

**Evolutionary conserved auxin response**
[Publication](https://www.biorxiv.org/content/10.1101/2022.11.25.517951v1)

_Project status Completed._

This project started as a follow up on the "Rapid auxin mediated phosphorylation response" project. Since many of the canonical transcriptional auxin (plant hormone) signalling components are not conserved throughout the green plant lineage, quantitiave phosphoproteomics datasets on representative green lineage plant species were generated to dissect a rapid evolutionary conserved response. To allow **cross dataset comparison**, repsective publicly available proteomes were used to generate orhtogroupings across the used green organisms. Using **R** and **Python** data was wrangled and analyzed on conserved phosphorylation patterns. This analysis identified an evolutionary conserved rapid auxin response, key players and physiological outcomes.

![Auxinevo](/assets/img/Evoauxin.jpg)

**Rapid auxin mediated phosphorylation response**
[Publication](https://www.biorxiv.org/content/10.1101/2022.11.25.517949v1)

_Project status Completed._
  
To gain a deeper insight into rapid and dynamic auxin (plant hormone) mediated signaling quantitative phosphorylation time series data was generated. Using **R** and **Python** this data was wrangled and explored to unravel key signaling hubs using the **Minardo-Model, distance metrics and network analysis**. The analysis identified a key player in rapid auxin mediated phosphorylation based signaling which provided a bases for further funding and follow up studies. The results have been made available through a **R Shiny** app [RShinyapp](https://weijerslab.shinyapps.io/AuxPhos/).

![Auxintimeseries](/assets/img/Auxintimeseries.jpg)

**Linking auxin receptors to rapid proteome responses**
[Publication](https://www.nature.com/articles/s41586-022-05187-x)
[Publication](https://www.nature.com/articles/s41586-021-04037-6)

_Project status Completed._

A valuable link was provided for our collaborators by collecting and analyzing phosphoproteomics datasets on auxin receptor mutants. These datasets were wrangled, mined and compared using **R**. The outcomes lead to new insights into auxin perception and signaling in A.Thaliana. 

![Auxinreceptor](/assets/img/Auxin receptor.jpg)

**Various projects**

![variousprojects](/assets/img/Various projects.jpg)

In various projects I **consulted** on **technical** and **data analytics** approaches to provide knowledgeable insights into stakeholders questions:
 1. Identifying targets of various kinases and phosphatases in mutant A.Thaliana backgrounds. Skills used: LC-MS/MS, R and Python
 2. Identifying targets of a kinase that causes change in color of Petunia. Skills used: LC-MS/MS, R 
 3. Describing the molecular changes in a mutant causing failure of reproduction in the fungus Schizophyllum commune. Skills used: LC-MS/MS, R and Python
 4. Identifying targets and interactors of polarly localized proteins in C.Elegans. Skills used: LC-MS/MS, R
 5. Developing data analysis pipeline to identify interactors of polarly localized proteins in A.Thaliana and M.Polymorpha. Skills used: R and Python

## Publications
* Fontana M., Roosjen M., Garcia IC., van den Berg W.,Malfois M., Boer R., Weijers D., Hohlbein J. 2023.Cooperative action of separate interaction domains promotes high-affinity DNA binding of Arabidopsis thaliana ARF transcription factors. PNAS, 120 (11) e2219916120
* Roosjen M., Kuhn A., Mutte SK., Boeren S., Krupar P., Koehorst J., Fendrych M., Friml J., Weijers D. 2022.An ultra-fast, proteome-wide response to the plant hormone auxin. In submission, preprint available 
* Kuhn A., Roosjen M., Mutte SK., Dubey SM., Carrasco PC,. Monzer A., Kohchi T., Nishihama R.,Fendrych M.,Friml J., Sprakel J., Weijers D., 2022. A RAF-like kinase mediates a deeply conserved,ultra-rapid auxin response. In submission, preprint available https://doi.org/10.1101/2022.11.25.517951
* Friml, J., Gallei, M., Gelová, Z.,Johnson A., Mazur E., Monzer A., Rodriguez,. Roosjen M.,Verstraeten, I.,Živanović, B.D et al. 2022. ABP1–TMK auxin perception for global phosphorylation and auxin canalization. Nature, 609, 575–581
* van Andel E, Roosjen M, Van der Zanden S, Lange S, Weijers D, Smulders MMJ, Savelkoul HFJ,Zuilhof H, Tijhaar EJ. 2022. Highly Specific Protein Identification by Immunoprecipitation–Mass Spectrometry Using Antifouling Microbeads, ACS Appl. Mater. Interfaces, 14, 20, 23102–23116
* Li, L., Verstraeten, I., Roosjen, M., Takahashi, K., Rodriguez, L., Merrin, J., Chen, J.,Shabala, L., Smet, W., Ren, H., et al. 2021 Cell surface and intracellular auxin signalling for H+fluxes in root growth. Nature, 599, 273–277
* Roosjen M. 2020, Proteomics perspective on auxin biology. PhD Thesis. 9789463952705
* Roosjen M, Paque S. Weijers D. 2018. Auxin response factors: output control in auxin biology Journal of experimental botany, 69, 2, pp179–188
* Smits AH, Borrmann A, Roosjen M, van Hest JCM, Vermeulen M. 2016. Click-MS: tagless protein enrichment using bioorthogonal chemistry for quantitative proteomics ACS Chemical Biology, 11, 12,pp 3245–3250
* Roosjen M, McColl B, Kao B,Gearing LJ, Blewitt ME, Vadolas J. 2013. Transcriptional regulators Myband BCL11A interplay with DNA methyltransferase 1 in developmental silencing of embryonic and fetal β-like globin genes. FASEB J. 28, 4, pp 1610-1620
